.github/workflows/greet_first_time_contributor.yml

```diff
@@ -0,0 +1,50 @@
+name: Greet first time contributor
+
+on:
+  issues:
+    types: [opened]
+  pull_request:
+    types: [opened, closed]
+
+jobs:
+  check_for_first_interaction:
+    runs-on: ubuntu-latest
```

Comment:
Please use `ubuntu-22.04` here as do the rest of the workflows.

We do not want any surprises when GH decided to change the `latest` version in the future.